src/pages/EmergencyViewPage/EmergencyViewPage.tsx

```diff
@@ -0,0 +1,281 @@
+import { useSelector } from "hooks/useSelector";
+import { useUser } from "hooks/useUser";
+import { useVenueId } from "hooks/useVenueId";
+import useVenueScheduleEvents from "hooks/useVenueScheduleEvents";
+
+import { updateTheme } from "pages/VenuePage/helpers";
+
+import WithNavigationBar from "components/organisms/WithNavigationBar";
+
+import { LoadingPage } from "components/molecules/LoadingPage";
+import { ScheduleEventSubListNG } from "components/molecules/ScheduleEventListNG/ScheduleEventSubListNG";
+
+import EmergencyViewRoom from "./EmergencyViewRoom";
+
+import "./EmergencyViewPage.scss";
+
+export const emptyPersonalizedSchedule = {};
+
+const tabs = [
+  { id: 0, title: "Spaces" },
+  { id: 1, title: `What's on` },
+];
+
+export const EmergencyViewPage: React.FC = () => {
+  const [selectedTab, updateTab] = useState(0);
+  const venueId = useVenueId() || "";
+  useConnectCurrentVenue();
+
+  const venue = useSelector(currentVenueSelectorData);
+  const { userWithId } = useUser();
+  const userEventIds =
+    userWithId?.myPersonalizedSchedule ?? emptyPersonalizedSchedule;
+
+  const {
+    descendantVenues,
+    dayDifference,
+    liveAndFutureEvents,
+    firstScheduleDate,
+  } = useVenueScheduleEvents({
+    venueId,
+    userEventIds,
+  });
+
+  const venueRequestStatus = useSelector(
+    (state) => state.firestore.status.requested.currentVenue
+  );
+  const redirectUrl = venue?.config?.redirectUrl ?? "";
+  const { hostname } = window.location;
+
+  useEffect(() => {
+    if (redirectUrl && redirectUrl !== hostname) {
+      window.location.hostname = redirectUrl;
+    }
+  }, [hostname, redirectUrl]);
+
+  dayjs.extend(advancedFormat);
+
+  useEffect(() => {
+    if (!venue) return;
+
+    // @debt replace this with useCss?
+    updateTheme(venue);
+  }, [venue]);
+
+  const weekdays = useMemo(() => {
+    return range(dayDifference)
+      .map((dayIndex) => {
+        const day = addDays(firstScheduleDate, dayIndex);
+
+        const dailyEvents = liveAndFutureEvents.filter(
+          isEventWithinDateAndNotFinished(day)
+        );
+
+        const eventsFilledWithPriority = dailyEvents
+          .map((event) => ({
+            ...event,
+            orderPriority: event.orderPriority ?? 0,
+          }))
+          .sort(eventTimeAndOrderComparator);
+
+        if (!dailyEvents.length) {
+          return null;
+        }
+
+        return (
+          <div
+            className="EmergencyView__weekdays__column"
+            key={day.toISOString()}
+          >
+            <ScheduleEventSubListNG
+              events={eventsFilledWithPriority}
+              title={`Events on ${formatDateRelativeToNow(day)}`}
+              isShowBookmark={false}
+              isShowImage={false}
+              isShowDayDate={false}
+            />
+          </div>
+        );
+      })
+      .filter((day) => !!day);
+  }, [dayDifference, liveAndFutureEvents, firstScheduleDate]);
```

Comment:
Can be extracted into a separate utility component (including wrapping `<ul>` tag) for the `EmergencyViewPage`